config/application.rb

```diff
@@ -32,6 +32,7 @@ class Application < Rails::Application
 
     config.after_initialize do
       require "extends/controllers/decidim/devise/sessions_controller_extends"
+      require "extends/controllers/decidim/budgets/projects_controller_extends"
```

Comment:
I prefer to extend in the dedicated file `config/initializers/extends.rb` but I agree I can't explain why it causes a context error on Decidim Awesome